e2e/ansible/roles/k8s-openebs-cleanup/tasks/main.yml

```diff
@@ -26,21 +26,6 @@
   delay: 10
   retries: 6
 
-- name: Delete the openebs pre-release features
-  shell: source ~/.profile; kubectl delete -f {{ ansible_env.HOME }}/{{ openebs_pre_release_alias }}
-  args:
-    executable: /bin/bash
-  ignore_errors: yes
-
-- name: Confirm pre-release features have been deleted
```

Comment:
cast .. clean up check should be there. As part of openebs operator clean up, when openebs namespace is deleted, the cast yamls / objects should be deleted.

Reply:
Yes @kmova Added this cleanup check task post openebs-operator deletion.

Reply:
Yes @kmova Added this cleanup check task post openebs-operator deletion.